renovate.json

```diff
@@ -28,6 +28,10 @@
       {
         "matchPackageNames": ["Django"],
         "allowedVersions": "=3.2"
+      },
+      {
+        "matchPackageNames": ["wagtail"],
+        "allowedVersions": "<4.2"
```

Comment:
I think we decided to go with the `=` as above, so it should be `=4.1`. But also that's not suuperimportant to me.